client/src/components/AppHeader/AppHeader.scss

```diff
@@ -51,31 +51,20 @@
         }
 
         .search-field {
-            width: 35px;
+            width: 75px;
             transition: width 0.5s;
             border: 0;
             outline: none;
-            border-bottom-right-radius: 50px;
-            border-top-right-radius: 50px;
+            border-radius: 50px;
             box-shadow: none;
+            text-decoration: #e9ece5;
```

Comment:
this property gets coverted to 
    text-decoration-line: initial;
    text-decoration-style: initial;
    text-decoration-color: #e9ece5;

Maybe as variant better just point   text-decoration-color: #e9ece5; ? Win is 1 css rule, not 3.

Reply:
Thank you, I will fix.

Reply:
Made a new commit with amendments.